app/src/main/java/io/pslab/communication/ScienceLab.java

```diff
@@ -156,9 +150,24 @@ private void initializeVariables() {
         squareWaveFrequency.put("SQR2", 0.0);
         squareWaveFrequency.put("SQR3", 0.0);
         squareWaveFrequency.put("SQR4", 0.0);
+        if (CommunicationHandler.PSLAB_VERSION == 6) {
+            dacChannels.put("PCS", new DACChannel("PCS", new double[]{0, 3.3}, 0, 0));
+            dacChannels.put("PV3", new DACChannel("PV3", new double[]{0, 3.3}, 1, 1));
+            dacChannels.put("PV2", new DACChannel("PV2", new double[]{-3.3, 3.3}, 2, 0));
+            dacChannels.put("PV1", new DACChannel("PV1", new double[]{-5., 5.}, 3, 1));
+        } else {
```

Comment:
Same question as earlier: Should we check for version 5 here and add an `else` block for the handling of unexpected versions?